XPNet.v3.ncrunchsolution.user

```diff
@@ -0,0 +1,5 @@
+﻿<SolutionConfiguration>
```

Comment:
Ditto .user file - remove from git and add to .gitignore instead.

Reply:
In fact I found these user files also in the original repository and removed all of them. Added to .gitignore